Sources/Controllers/2.0/Settings/OAImportDuplicatesViewController.mm

```diff
@@ -348,21 +345,55 @@ - (void) setupBottomViewMultyLabelButtons
     self.primaryBottomButton.hidden = NO;
     self.secondaryBottomButton.hidden = NO;
     
-    [self setToButton: self.secondaryBottomButton firstLabelText:OALocalizedString(@"keep_both") firstLabelFont:[UIFont systemFontOfSize:15 weight:UIFontWeightSemibold] firstLabelColor:UIColorFromRGB(color_primary_purple) secondLabelText:OALocalizedString(@"keep_both_desc") secondLabelFont:[UIFont systemFontOfSize:13] secondLabelColor:UIColorFromRGB(color_icon_inactive)];
+    NSDictionary *secondaryButtonParams = @{
+        @"firstLabelText": OALocalizedString(@"keep_both"),
```

Comment:
Couldn't you simply setup most of these parameters in the xib and only apply the localization in ```applyLocalization``` method?